less/select.less

```diff
@@ -15,6 +15,7 @@
 // control options
 @select-input-bg:                   #fff;
 @select-input-bg-disabled:          #f9f9f9;
+@select-input-bg-focus:             #fff;
```

Comment:
Please make this default to `@select-input-bg` in case that has been customised, otherwise this would be a breaking change.